daemon/create.go

```diff
@@ -78,6 +78,21 @@ func (daemon *Daemon) containerCreate(params types.ContainerCreateConfig, manage
 	return containertypes.ContainerCreateCreatedBody{ID: container.ID, Warnings: warnings}, nil
 }
 
+func setContainerIDEnv(container *container.Container, cidenv string) error {
+	if cidenv == "" {
+		return nil
+	}
+	// check if we already have an env var like that
+	for _, env := range container.Config.Env {
```

Comment:
Where are the rest of env variables set? In `daemon.newContainer`?

Reply:
yep, here: https://github.com/balena-os/balena-engine/blob/6ac0449ca14cc2871bc498174f9fa891c9437d54/daemon/container.go#L156

Here is the type that this corresponds to: https://github.com/balena-os/balena-engine/blob/6ac0449ca14cc2871bc498174f9fa891c9437d54/api/types/container/config.go#L54